plugins/lime-plugin-pirania/src/components/createVoucherForm.js

```diff
@@ -20,8 +22,12 @@ const CreateVoucherForm = ({ submitVoucher }) => {
 			[field]: value
 		});
 	};
+	const createVoucher = (e) => {
+		setLoading(true);
+		submitVoucher(e, input);
+	};
 	return (
-		<form class={style.createForm} onSubmit={(e) => submitVoucher(e, input)}>
+		<form class={style.createForm} onSubmit={(e) => createVoucher(e)}>
```

Comment:
We are using [react-hook-form](https://react-hook-form.com/) to build forms, handle validation, etc.

There are some screens in the project already using it, did you come across any of them? Do you think it would simplify the code for this form?